prow/cluster/build/200-serviceaccounts.yaml

```diff
@@ -11,5 +11,41 @@ metadata:
 kind: ServiceAccount
 apiVersion: v1
 metadata:
-  name: "boskos"
+  annotations:
+    iam.gke.io/gcp-service-account: prow-job@knative-tests.iam.gserviceaccount.com
+  name: boskos
+  namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: prow-job@knative-tests.iam.gserviceaccount.com
+  name: test-runner
+  namespace: test-pods
+---
+# we need to drop the annotation on this service account after the istio prowgen is fixed.
+# There is a bug where a service account can't specified in requirements presets :(
```

Comment:
I saw your messages on Slack but I forgot to reply, sorry for that..
I think it'll be great if `PodSpec` is part of the `RequirementPreset`. I implemented the `mergeRequirement` function before I used the `github.com/imdario/mergo` library. I haven't looked into the details, but probably using `mergo` is easy for implementing `mergePodSpec` since Kubernetes is also using this library.